### PR TITLE
muon real data analysis

### PR DESCRIPTION
**Pull request by pillera**
Not optimized preliminary script to analyze real data runs extracting features relevant for muon preselection.

### CODE REVIEW

File: notebooks/Read_and_plot_LST_Data.ipynb [hunk 1]
```diff
@@ -812,7 +812,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
+<<<<<<< HEAD
    "version": "3.7.3"
+=======
+   "version": "3.6.5"
+>>>>>>> 71399bf6e2af2638cd48a61813380c1583a879ea
```
Comment (vuillaut) [inline notebooks/Read_and_plot_LST_Data.ipynb]: Please clean-up this.
(in this case I don't think this notebook needs to be included)

File: notebooks/Calculate_sensitivity_eventio.ipynb [hunk 2]
```diff
@@ -133,7 +133,11 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
    "execution_count": 8,
+=======
+   "execution_count": 10,
+>>>>>>> 71399bf6e2af2638cd48a61813380c1583a879ea
    "metadata": {},
```
Comment (vuillaut) [inline notebooks/Calculate_sensitivity_eventio.ipynb]: Please clean-up the notebook
ps: it does not seem that you made changes to the notebook that need to be pushed, maybe you just want to completely remove it from the PR

File: notebooks/Step_by_Step_dl0_to_dl1.ipynb [hunk 3]
```diff
@@ -1572,9 +1572,7 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": [
-    ""
-   ]
+   "source": []
```
Comment (vuillaut) [inline notebooks/Step_by_Step_dl0_to_dl1.ipynb]: Same or this notebook, no real changes here ?

File: notebooks/read_LST1_data.ipynb [hunk 4]
```diff
@@ -1265,7 +1265,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.4"
+   "version": "3.6.5"
```
Comment (vuillaut) [inline notebooks/read_LST1_data.ipynb]: Same comment

**Comment by rlopezcoto** [comment]
@vuillaut thanks, I can take care of the script.

File: notebooks/Calculate_sensitivity_eventio.ipynb (hunk 2 above)
Comment (pillera) [reply notebooks/Calculate_sensitivity_eventio.ipynb]: yes I will remove it, I thought I added only my new script in this PR.

**Comment by rlopezcoto** [comment]
Hi @pillera, now the PR would eliminate all notebooks and muon script. I am not sure if this is what you wanted to do :D 
Shall we close this PR and you can open a new one in which you only commit the script?

**Comment by pillera** [comment]
> Hi @pillera, now the PR would eliminate all notebooks and muon script. I am not sure if this is what you wanted to do :D
> Shall we close this PR and you can open a new one in which you only commit the script?

Okay, I don't know why it ended up like this, I thought I only removed the changes on the notebooks, not the whole files, something went wrong...
I wil close it, and I'll try again.

**Comment by rlopezcoto** [comment]
don't worry about this, it is not a problem ;-)

**Pull request closed without merging**